account-specific.tf

```diff
@@ -10,9 +10,10 @@ resource "aws_kms_key" "specific_remote_state_backend" {
 }
 
 resource "aws_kms_alias" "specific_state_backend" {
-  count         = length(var.account_arns)
-  name          = "alias/${var.name_prefix}-remote-state-backend-${replace(var.account_arns[count.index], local.account_from_arn, "$5")}${var.name_suffix}"
-  target_key_id = aws_kms_key.specific_remote_state_backend[*].key_id[count.index]
+  count = length(var.account_arns)
+  name  = "alias/${var.name_prefix}-remote-state-backend-${replace(var.account_arns[count.index], local.account_from_arn, "$5")}${var.name_suffix}"
+  # tflint-ignore: terraform_deprecated_index ignored because terraform will threat it as list
+  target_key_id = aws_kms_key.specific_remote_state_backend.*.key_id[count.index]
```

Comment:
Does this change affect pre-existing remote state buckets?  In other words, is it a breaking change or does the interpolation continue to work as expected?

Reply:
This is just for ignoring and "error" because tflint is thinking it is a list, but the code is OK and unchanged